regulations/static/regulations/js/source/views/comment/comment-review-view.js

```diff
@@ -58,11 +55,13 @@ var CommentReviewView = Backbone.View.extend({
       dataType: 'json'
     });
     $xhr.then(this.previewSuccess.bind(this));
+    this.previewLoading = true;
+    this.render();
   },
 
   previewSuccess: function(resp) {
-    this.mode = 'submit';
-    this.previewUrl = resp.url;
+    window.location = resp.url;
```

Comment:
Would it be possible to create a new tab instead? There's certainly an argument that the user expects to be "taken" to the file (despite Chrome's helpful stay-on-page-anyway approach), but it feels a little shady to do a `window.location`